apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -269,7 +218,12 @@ const ProjectCard = React.createClass({
           }
         </div>
 
-        {this.renderActionBox(this.checkIfPublished())}
+        {this.state.actionsOpen &&
+          <ProjectActionBox
+            isPublished={this.props.projectData.publishedToPublic}
+            style={{position: 'absolute', marginLeft: 10, marginTop: '-10px'}}
```

Comment:
nit: i would put this style in our `styles` object (in general, i would advocate for doing so any time you have more than one style field)
also, you use a number for marginLeft but a string for marginTop?